app/scripts/directives/oscPersistentVolumeClaim.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
+angular.module("openshiftConsole")
+  /**
+   * Widget for entering PVC information
+   *
+   * model:
+   *   The model for the input. The model will either use or add the following keys:
+   *     {
+   *       name: "",
+   *       accessMode: "",
+   *       Capacity: ""
+   *     }
+   *
+   */
+  .directive("oscPersistentVolumeClaim", function(){
+    return {
+      require: '^form',
+      restrict: 'E',
+      scope: {
+        route: "=model",
```

Comment:
Copied and pasted from the route code and forgot to update - I will change that to claim